lib/samson/console_extensions.rb

```diff
@@ -0,0 +1,19 @@
+module Samson
+  module ConsoleExtensions
+    def login(role = Role::SUPER_ADMIN)
+      user = User.where(role_id: role.id).last!
```

Comment:
Maybe if you just name it `login_as_role` then it would be more clear what you're doing.

And add some docs on the purpose of the method.  When I read it, I thought the idea was that if you did something like trigger a deployment, it would log the correct user as doing it.  For compliance reasons it's important that we keep track of who's doing what, so I want to be careful about mechanism that allow impersonation.